AzureMonitorAgent/agent.py

```diff
@@ -1146,8 +1146,22 @@ def find_package_manager(operation):
     for rpm_dist in rpm_set:
         if dist.startswith(rpm_dist):
             PackageManager = "rpm"
-            # Same as above.
+            # Same as above; need to force.
             PackageManagerOptions = "--force"
+
+            # If FIPS is enabled and RPM version is 4.14 or higher, we need to apply the below workaround
+            try:
```

Comment:
I would suggest to have fips in seperate commit and PR as it is easy to revert or look back in history if issues happen with the changes. Also CIS benchmark related change seems unrelated.

Reply:
Sure, I have added as a separate PR #1611